app/controllers/declaration/SealController.scala

```diff
@@ -80,12 +80,9 @@ class SealController @Inject()(
     )
 
   private def badRequest(formWithErrors: Form[Seal], cachedSeals: Seq[Seal])(implicit request: JourneyRequest[_]) = {
-    val declaration = exportsCacheService.get(request.declarationId)
-    declaration.map(_.flatMap(_.transportDetails)).flatMap { data =>
-      declaration.map(_.map(_.seals)).map { seals =>
-        BadRequest(sealPage(formWithErrors, seals.getOrElse(Seq.empty), data.fold(false)(_.container)))
-      }
-    }
+    val d = request.cacheModel.transportDetails
```

Comment:
I prefer to have better naming than `d` or `s`. I know the scope is small, but it'll be easier to read when you will have whole word

Reply:
Fixed in later commit

Reply:
Still `d` and `s`